pom.xml

```diff
@@ -667,7 +667,7 @@ flexible messaging model and an intuitive client API.</description>
       <dependency>
         <groupId>com.github.ben-manes.caffeine</groupId>
         <artifactId>caffeine</artifactId>
-        <version>2.3.3</version>
+        <version>2.6.2</version>
```

Comment:
We also need to update the license file with the list of binaries: `distribution/server/src/assemble/LICENSE.bin.txt`

Reply:
Yes, the validation check for the Java job otherwise is failing :)